documentation/guides/rust/get-started/05-secure-channel/README.md

```diff
@@ -38,19 +38,22 @@ Add the following code to this file:
 // It then runs forever waiting for messages.
 
 use hello_ockam::Echoer;
+use ockam::access_control::AllowAll;
 use ockam::authenticated_storage::InMemoryStorage;
 use ockam::identity::{Identity, TrustEveryonePolicy};
 use ockam::{vault::Vault, Context, Result, TcpTransport};
 
 #[ockam::node]
 async fn main(ctx: Context) -> Result<()> {
-    ctx.start_worker("echoer", Echoer).await?;
+    ctx.start_worker("echoer", Echoer, AllowAll, AllowAll).await?;
 
     // Initialize the TCP Transport.
     let tcp = TcpTransport::create(&ctx).await?;
 
     // Create a TCP listener and wait for incoming connections.
-    tcp.listen("127.0.0.1:4000").await?;
+    // Use port 4000, unless otherwise specified by command line argument.
+    let port = std::env::args().nth(1).unwrap_or_else(|| "4000".to_string());
+    tcp.listen(format!("127.0.0.1:{port}")).await?;
```

Comment:
Let's revert the port input change back to the way they was. We want examples to stay focused on teaching the core idea we're introducing.

Reply:
The problem is that these examples were not checked on CI, I added that check. Fixed ports can cause problems during parallel runs. Thoughts?

Reply:
🤔 Maybe we can tell the test runner to run the tests that involve ports in sequence.

Reply:
Maybe, but as far as I remember - using posts 3000 4000 is road to random failures

Reply:
We can make some other, less likely to collide, ports the hardcoded values - maybe 4441, 4442, 4443.

That would be a good change, we've come across similar collisions with macos as well.

But we should make it, consistently, across all examples and diagrams.
There is this PR that didn't change diagrams https://github.com/build-trust/ockam/pull/2700

Reply:
@mrinalwadhwa I just noticed that our get started guide for transports (that precedes this one) already uses port numbers from cli arguments. Therefore I don't get why doing the same in this guide is bad

Reply:
It's bad because this adds noise to the example that takes attention away from the main ideas we're trying to teach here. Think of a non rust developer or not a developer at all reading this.

If the transport example uses it, it must've missed my check .. let's change that to hardcoded values as well

Reply:
I reverted get started examples to the simpler state and created an issue for further improvement instructions. I think we should not prevent this PR from landing because of that issue
V

Reply:
sounds good 👍